apps/api/src/controllers/admin.controller.ts

```diff
@@ -323,6 +324,10 @@ export function makeAdminController(services: ServiceRegistry) {
                 },
           });
 
+        const activeCompetition =
```

Comment:
can we just use the `competition` that was returned above rather than re-querying the database here?

Reply:
Yea it's kinda annoying because `services.competitionService.startOrCreateCompetition()` returns a competition object that doesn't contain the `crossChainTradingType` property which is needed. `services.competitionService.getActiveCompetition()` does return that property. This is the result of a (necessary and good) abstraction we made in the db where the core competitions table is general for any type of competition. We have another table, `trading_competitions` which extends that to add trading-related things to competitions. The issue is that we haven't built a clean API on top of those table that allows you to properly and efficiently interact with those different types of competitions. Things are all over the place at the moment.

Reply:
okay, that's fine, we can leave it for now and revisit later